### PR TITLE
Feature/nwps 821 a to z listing

### PR DESCRIPTION
**Pull request by Raq-HEE**
Hi @sankar-shunmuga / @adaMiclos - can you give this a review please.

I've created this as a standalone rather than an extention of Listing page because a) majority of fields weren't required from Listing page and b) majority of methods from ListingPageComponent weren't being used.

### CODE REVIEW

**Comment by sankar-shunmuga** [comment]
Raq, I tried with the following pages from CMS folders.

**Check Box visible & functionality works as expected**
1. Listing Page
2. A to Z Test Content

Check Box visible & functionality not worked when checked
1. Standard Content Pages ( Introduction)
2. Landing Pages ( About)

**Review by sankar-shunmuga — CHANGES_REQUESTED** [review]
Hi Raq, Please see my three comments.

File: site/components/src/main/java/uk/nhs/hee/web/beans/LandingPage.java [hunk 1]
```diff
@@ -1,38 +1,43 @@
...
+        return getLinkedBean("hee:heroImage", ImageSetWithCaption.class);
+    }
+
+    @HippoEssentialsGenerated(internalName = "hee:logoGroup")
+    public HippoBean getLogoGroup() {
+        return getLinkedBean("hee:logoGroup", HippoBean.class);
+    }
+
+    @HippoEssentialsGenerated(internalName = "hee:boolean")
+    public Boolean getBoolean() {
```
[the remark this answers is not in the document]
Comment (Raq-HEE) [reply site/components/src/main/java/uk/nhs/hee/web/beans/LandingPage.java]: Done.

File: site/components/src/main/java/uk/nhs/hee/web/components/AToZListingPageComponent.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+import java.util.*;
+
+/**
+ * Base component for A to Z Listing Page.
+ */
+@ParametersInfo(type = AToZPageComponentInfo.class)
+public class AToZListingPageComponent extends EssentialsDocumentComponent {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AToZListingPageComponent.class);
+
+    public void doBeforeRender(final HstRequest request, final HstResponse response) {
```
[the remark this answers is not in the document]
Comment (Raq-HEE) [reply site/components/src/main/java/uk/nhs/hee/web/components/AToZListingPageComponent.java:36]: Discussed previously with Selva and the bloomreach api for search should be optimised. So for this iteration, we will check performance and can add caching in a later iteration if needed.

**Review by sankar-shunmuga — APPROVED** [review]
Thanks for the changes

**Pull request merged**